specification/indices/put_data_stream_settings/IndicesPutDataStreamSettingsRequest.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IndexSettings } from '@indices/_types/IndexSettings'
+import { RequestBase } from '@_types/Base'
+import { Indices } from '@_types/common'
+import { Duration } from '@_types/Time'
+
+/**
+ * Update data stream settings.
+ * This API can be used to override settings on specific data streams. These overrides will take precedence over what
+ * is specified in the template that the data stream matches. To prevent users from getting into an invalid state,
+ * only certain settings are allowed. If the can reasonably be applied to all backing indices, it is applied to all
+ * backing indices. Otherwise, it will be applied only whenever the data stream is next rolled over.
+ * @rest_spec_name indices.put_data_stream_settings
+ * @availability stack stability=stable visibility=feature_flag feature_flag=logs_stream
+ * @availability serverless stability=stable visibility=feature_flag feature_flag=logs_stream
+ * @index_privileges manage
+ * @doc_id indices-put-data-stream-settings
```

Comment:
Ditto re defining the tag explicitly.
```suggestion
 * @doc_id indices-put-data-stream-settings
 * @doc_tag data stream
```